tasks/comment/InfracostComment/tests/mock-helper.ts

```diff
@@ -0,0 +1,59 @@
+import MockAnswer = require('azure-pipelines-task-lib/mock-answer');
+import MockRun = require('azure-pipelines-task-lib/mock-run');
+import path = require('path');
+
+export default function setupRunner(opts: any): MockRun.TaskMockRunner {
```

Comment:
would prefer having an explicit type here to know the `opts` but not a biggie as it's a test helper.

Reply:
Agree, but it's not trivial to extract it from the compost, so I decided to not spend time more time on it 🤷‍♂️.

Reply:
👍

Reply:
I got what you mean, added an interface 👍 .